packages/ERTP/src/issuer.js

```diff
@@ -106,8 +106,12 @@ function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT) {
       },
       getCurrentAmount: () => purseLedger.get(purse),
       getAllegedBrand: () => brand,
-      makeDepositFacet: () => harden({ receive: purse.deposit }),
+      // eslint-disable-next-line no-use-before-define
+      getDepositFacet: () => depositFacet,
     });
+
+    const depositFacet = harden({ receive: purse.deposit });
```

Comment:
why not move this up so we don't have to use the eslint-disable?

Reply:
If it's before purse, then purse needs the eslint-disable. If it's inside purse, then it's visible to purse holders. I could have used let, but that still needs an assignment after the purse is defined.

This seemed better than any of those possibilities. Did I miss another approach?

Reply:
Oh, gotcha. Sorry, missed the circular reference. This looks fine